### ISSUE TITLE
Sinks use unlocalized name instead of name in lang file

### ISSUE THREAD

**Issue by darthvader45** [MatrexsVigil/harvestcraft#163]
I just noticed that the sinks are called tile.sink.name.  Might want to fix this, so that each sink has its own unique name.

**Comment by FledgeXu** [comment]
yes,it's seems that the reason of the Pam's HarvestCraft not support the multi-language. such as Chinese.

**Comment by FledgeXu** [comment]
I‘m also curious that when will the HarvestCraft support the multi-language.
